specification/glossary.md

```diff
@@ -60,6 +60,10 @@ A specification-defined categorical attribute that provides context or categoriz
 
 Cost inclusive of the impacts of all reduced rates and discounts, augmented with the amortization of relevant purchases (one-time or recurring) paid to cover future eligible charges.
 
+<a name="glossary:exclusivebound"><b>Exclusive Bound</b></a>
+
+A Date/Time Format value that is not contained within the ending bound of a time period.
+
```

Comment:
```suggestion
<a name="glossary:exclusivebound"><b>Exclusive Date/Time Value</b></a>

A date/time value that is exclusive can be used to represent the end of a period where the period ends immediately before the specified date/time. An exclusive date/time is in contrast to an [inclusive date/time](#glossary:inclusivebound), which is commonly used to signify the start of a period.

```
Suggestion. Ignore if it doesn't improve. If accepted, the links to this glossary will need to be updated.